### ISSUE TITLE
Accessibility: A link contains no text.

### ISSUE THREAD

**Issue by StarWar** [huacnlee/social-share-button#134]
I've tested with WAVE extension for accessibility, the sharing links generated by social-share-button has no text for links, what should I do to remove these errors.
```
Empty link
A link contains no text.
```
It should have text/labels so that screen readers can read.
